comet/utils/redis.py

```
@@ -0,0 +1,215 @@
+            return False
+        except RedisError as e:
+            logger.error(f"Redis error during EXISTS for key {self._sanitize_key_for_logging(key)}: {e}")
+            return False
+
+    async def expire(self, key: str, ttl: int) -> bool:
+        if not self.is_connected():
+            return False
+        
+        try:
+            await self.redis.expire(key, ttl)
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)